contrib/kafka/config/500-controller.yaml

```diff
@@ -32,10 +32,12 @@ spec:
       containers:
       - name: manager
         image: github.com/knative/eventing-sources/contrib/kafka/cmd/controller
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
         env:
         - name: KAFKA_RA_IMAGE
           value: github.com/knative/eventing-sources/contrib/kafka/cmd/receive_adapter
+        - name: KAFKA_RA_IMAGE_PULL_POLLICY
```

Comment:
If that KAFKA_RA_IMAGE_PULL_POLLICY env variable is not present then IfNotPresent is used - I think ability to configure image pull policy for receive adapter is good? @dubee @matzew